tests/e2e/cucumber/features/app-provider/officeSuites.feature

```diff
@@ -26,6 +26,15 @@ Feature: Integrate with online office suites like Collabora and OnlyOffice
     And "Anonymous" should be able to edit content of following resource
       | resource         | type         | content                     |
       | OpenDocument.odt | OpenDocument | Edited OpenDocument Content |
+    # For seeing the content updating in real time on Alice's screen, Alice needs to have the file opened
+    And "Alice" opens the following file in Collabora
+      | resource         |
+      | OpenDocument.odt |
+    When "Anonymous" edits the following resource
+      | resource         | type         | content                     |
+      | OpenDocument.odt | OpenDocument | Edited OpenDocument Content |
```

Comment:
the content of the file seems to be same as in above step in line 26. Use different content for this step.
Or, maybe we can move the step `And "Alice" opens the following file in Collabora` above the step `And "Anonymous" should be able to edit content of following resource`, with this we would not need this step and shorten the user story.

Reply:
journey like this would be good here:
```feature
And "Alice" creates a public link of following resource using the sidebar panel
And "Alice" opens the following file in Collabora
And "Anonymous" opens the public link "Link"
And "Anonymous" unlocks the public link with password "%public%"
Then "Anonymous" should see the content "OpenDocument Content" in editor "Collabora"
When "Anonymous" edits the following resource
Then "Alice" should see the content "Edited OpenDocument Content" in editor "Collabora"
And "Alice" closes the file viewer
```